libs/blocks/chart/chart.js

```diff
@@ -122,7 +122,9 @@ export function processMarkData(series, xUnit) {
 }
 
 export async function fetchData(link) {
-  const resp = await fetch(link.href.toLowerCase());
+  const params = new URLSearchParams(window.location.search);
+  const cache = params.get('cache') === 'off' ? 'reload' : 'default';
+  const resp = await fetch(link.href.toLowerCase(), { cache });
```

Comment:
Since the exact same functionality is being used across multiple files, I think we should make this a utility function that is shared by the blocks in need. Making that update also makes it extendable to future blocks, and simplifies testing a bit as well. (You will only have to mock the search param to get code cov, and add a utility test)